engine/pom.xml

```diff
@@ -23,7 +23,7 @@
     <distributionManagement>
         <site>
             <id>site</id>
-            <url>http://validator.hibernate.org</url>
+            <url>http://hibernate.org/validator</url>
```

Comment:
>  that means the second one is your canonical representation. 

I am not sure about "canonical representation". Just because one URL is a redirect, does not mean it is less important than the URL it redirects to. The redirect gives Validator a more prominent URL and I think that is the whole purpose. There is no legacy involved here. There is no ORM, because ORM == hibernate.org (of course one could argue that such a redirect would make sense). Last but not least, there is search.hibernate.org and that's what's used in the Search POM.